src/components/AccountPage/AccountBalanceSummaryHeader.js

```diff
@@ -11,41 +9,24 @@ import type { Currency, Account } from '@ledgerhq/live-common/lib/types'
 import type { T } from 'types/common'
 
 import { saveSettings } from 'actions/settings'
-import { accountSelector } from 'reducers/accounts'
-import { counterValueCurrencySelector, selectedTimeRangeSelector } from 'reducers/settings'
 import type { TimeRange } from 'reducers/settings'
 
-import {
-  BalanceTotal,
-  BalanceSinceDiff,
-  BalanceSincePercent,
-} from 'components/BalanceSummary/BalanceInfos'
+import { BalanceTotal, BalanceSinceDiff, BalanceSincePercent } from 'components/BalanceInfos'
 import Box from 'components/base/Box'
 import FormattedVal from 'components/base/FormattedVal'
 import PillsDaysCount from 'components/PillsDaysCount'
 
-type OwnProps = {
+type Props = {
   isAvailable: boolean,
-  totalBalance: BigNumber,
-  sinceBalance: BigNumber,
-  refBalance: BigNumber,
-  accountId: string,
-}
-
-type Props = OwnProps & {
+  first: *,
+  last: *,
```

Comment:
Not sure I understand why it is `*` ? In this case we know what it contains

Reply:
basically because i was lazy :p